packages/salesforcedx-vscode/package.json

```diff
@@ -28,7 +28,7 @@
     "vscode:publish": "node ../../scripts/publish-vsix.js"
   },
   "categories": ["Extension Packs"],
-  "extensionDependencies": [
+  "extensionPack": [
```

Comment:
Don't we need to update vscode engine version as well ?

Reply:
That's a good question. Let me check. I somehow thought it was retroactively applied since this is an extension pack. This is just a key per my understanding.

Reply:
Confirmed on #562 that we do need to bump up to the latest version.
I think this is OK since our stats indicate that our customers are apt to upgrade to the latest version of VS Code quickly.